src/main/java/org/gitlab/api/GitlabAPI.java

```diff
@@ -2179,7 +2215,7 @@ public void deleteProjectMember(Integer projectId, Integer userId) throws IOExce
      * @throws IOException on gitlab api call error
      */
     public List<GitlabProjectMember> getNamespaceMembers(Integer namespaceId) throws IOException {
-        String tailUrl = GitlabNamespace.URL + "/" + namespaceId + GitlabProjectMember.URL;
+        String tailUrl = GitlabGroup.URL + "/" + namespaceId + GitlabProjectMember.URL;
```

Comment:
Should there be new methods such as `getGroupMembers` etc? Do you think it might cause confusion?

Reply:
The API doc already states that the methods fails when the namespace is not a group. I don't know how exactly it fails. but renaming the method to getGroupMembers() would make its purpose clear.